ZeroWasteFriendly/Scenes/Library/LibraryController.swift

```diff
@@ -41,13 +45,21 @@ private extension LibraryController {
 
     private func setupTableView() {
         tableView.register(cellType: LibraryTableViewCell.self)
-        tableView.contentInset.top = 7
-        tableView.contentInset.bottom = 7
+        tableView.contentInset.bottom = 15
     }
 
     private func setupNavigationBar() {
         title = R.string.localizable.libraryListTitle()
     }
+
+    private func setupCollectionView() {
+        let viewWidth = view.frame.width
```

Comment:
are you sure this view.frame.width is valid? I'd say it's taken from XIB, not from real device, because iti isi in viewDidLoad, when layouts are not calculated,

you could send updates to collectionviewframe on viewDidLayoutSubviews, to make sure it's valid